static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.tsx

```diff
@@ -149,31 +153,46 @@ export function WidgetPreviewContainer({
   dashboardFilters,
   dashboard,
   isWidgetInvalid,
-  translate,
+  updatedTranslate,
   isDraggable,
 }: {
   dashboard: DashboardDetails;
   dashboardFilters: DashboardFilters;
   isWidgetInvalid: boolean;
   isDraggable?: boolean;
-  translate?: Translate;
+  updatedTranslate?: WidgetDragPositioning;
```

Comment:
Can we call it something like `position`? Something about `updatedTranslate` feels weird because it makes me think there's another variable that represents the default translate.